Bottom: 692b9e2fc3e636a232a73a70f2f55b8bb20045f6
Top:    214861a6226558b7f140d89f26ad8b92b132149a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 12:53:53 +0100

hid/gtk: Turn on defines to debug a potential GTK-3.0 build


---

diff --git a/configure.ac b/configure.ac
index 71ab1c4..d2e2eeb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -770,6 +770,9 @@ $GTKGLEXT_PKG_ERRORS])]
 	GTKGLEXT_VER=`$PKG_CONFIG gtkglext-1.0 --modversion`
 	fi
 
+	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+#	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
+
 	;;
 
       gcode|nelma|png )
